### DISCUSSION TITLE
About detector

### DISCUSSION THREAD

**Discussion by 1234soda** [fangq/mcx#230] [Help]
Hello, can I set the detection range of the detector to a rectangle, if so, how to set it?

**Comment by fangq** [comment]
hi @1234soda, sorry for not noticing this post.

you can use `cfg.bc` to detect photons along the bounding box faces of the volume, see [this example](https://github.com/fangq/mcx/blob/master/mcxlab/examples/demo_bc_det.m). but it currently does not support rectangular detector shape.
